package.json

```
@@ -1,11 +1,14 @@
 {
   "name": "packagephobia",
   "version": "1.0.0",
+  "private": true,
   "description": "Find the cost of adding a new dependency to your project",
   "scripts": {
     "build": "cp -r static public",
     "prettier": "prettier \"./**/*.ts\" \"./**/*.tsx\" --write",
-    "test": "tsc && tape dist/test/**/*.js"
+    "tape": "tape \"dist/test/**/*.js\"",
```

```suggestion
    "test": "tape \"dist/test/**/*.js\"",
```

I think the GH Actions should be adjusted to run both `tsc` and `test` jobs.

Then there is no need for the third script that runs both.

---

This patch runs both in test.

>    "test": "npm run tsc && npm run tape"

---

The idea is to be able to run the individual scripts on demand. At least that's what I'm used to. Similarly to the prettier script.

---

I think more scripts make this overwhelming. If you want to run a build only, you can do `npx tsc`.

If you want to change this PR to only have `private: true` I would accept that.

---

I will revert those changes since I can't really run the project locally. `npx` and npm are different things and not interchangeable.